test/test_functional_tensor.py

```diff
@@ -73,28 +118,51 @@ def test_adjustments(self):
                 # F uses uint8 and F_t uses float, so there is a small
                 # difference in values caused by (at most 5) truncations.
                 max_diff = (ft_img - f_img).abs().max()
+                max_diff_scripted = (sft_img - f_img).abs().max()
                 self.assertLess(max_diff, 5 / 255 + 1e-5)
+                self.assertLess(max_diff_scripted, 5 / 255 + 1e-5)
                 self.assertTrue(torch.equal(img, img_clone))
 
     def test_rgb_to_grayscale(self):
+
+        @torch.jit.script
+        def script_rgb_to_grayscale(input):
+            # type: (Tensor) -> Tensor
+            return F_t.rgb_to_grayscale(input)
         img_tensor = torch.randint(0, 255, (3, 16, 16), dtype=torch.uint8)
         img_tensor_clone = img_tensor.clone()
         grayscale_tensor = F_t.rgb_to_grayscale(img_tensor).to(int)
         grayscale_pil_img = torch.tensor(np.array(F.to_grayscale(F.to_pil_image(img_tensor)))).to(int)
         max_diff = (grayscale_tensor - grayscale_pil_img).abs().max()
         self.assertLess(max_diff, 1.0001)
         self.assertTrue(torch.equal(img_tensor, img_tensor_clone))
+        # scriptable function test
+        grayscale_script = script_rgb_to_grayscale(img_tensor).to(int)
+        self.assertTrue(torch.equal(grayscale_script, grayscale_tensor))
 
     def test_center_crop(self):
+
+        @torch.jit.script
+        def script_center_crop(input, output_size):
```

Comment:
Instead of creating a new function everytime, can't we just do something like
```python
scripted = torch.jit.script(F_t.center_crop)
self.assertTrue(torch.equal(scripted(tensor), F_t.center_crop(tensor)))
```

Reply:
Sure, this would be nice and concise.